src/RfbSocket.cpp

```diff
@@ -7,7 +7,12 @@
 #include <qtcpsocket.h>
 #include <qrect.h>
 
+#ifdef WIN32
+#include "WinSock2.h"
+#pragma comment(lib, "Ws2_32.lib")
```

Comment:
The include should be needed for ntoh/hton (l/s) calls. As the network byte order is "defined to always be big-endian" we probably can use qFromBigEndian/qToBigEndian instead ?

Reply:
Sounds good

Reply:
Changed the implementation to use qFromBigEndian/qToBigEndian.